packages/components/src/internal/components/chart/Chart.tsx

```diff
@@ -44,33 +43,33 @@ export class Chart extends React.Component<Props, State> {
         this.handleResize = debounce(this.handleResize.bind(this), 250);
     }
 
-    componentDidMount() {
-        $(window).on('resize', this.handleResize);
+    componentDidMount(): void {
+        window.addEventListener('resize', this.handleResize);
         this.getChartConfig();
     }
 
-    componentWillUnmount() {
-        $(window).off('resize', this.handleResize);
+    componentWillUnmount(): void {
+        window.removeEventListener('resize', this.handleResize);
     }
 
-    shouldComponentUpdate() {
+    shouldComponentUpdate(): boolean {
         return false;
     }
 
-    handleResize(e) {
+    handleResize(): void {
         this.renderChart();
     }
 
-    getPlotElement() {
-        return $('#' + this.state.divId);
+    getPlotElement(): HTMLDivElement {
+        return document.querySelector('#' + this.state.divId);
```

Comment:
nit: Could also be `document.getElementById(this.state.divId)`. Potato/potato.

Reply:
Using that does not allow the return type on this method to be HTMLDivElement for some reason, so it kind of needs to be querySelector.